core/src/emr-eks-platform/emr-eks-cluster-helpers.ts

```diff
@@ -72,37 +72,6 @@ export function eksClusterSetup(cluster: EmrEksCluster, scope: Construct, eksAdm
     }
   });
 
-  //Create service account for ALB and install ALB
-  const albPolicyDocument = PolicyDocument.fromJson(IamPolicyAlb);
-  const albIAMPolicy = new Policy(
-    scope,
-    'AWSLoadBalancerControllerIAMPolicy',
-    { document: albPolicyDocument },
-  );
-
-  const albServiceAccount = cluster.eksCluster.addServiceAccount('ALB', {
-    name: 'aws-load-balancer-controller',
-    namespace: 'kube-system',
-  });
-  albIAMPolicy.attachToRole(albServiceAccount.role);
-
-  const albService = cluster.eksCluster.addHelmChart('ALB', {
-    chart: 'aws-load-balancer-controller',
-    repository: 'https://aws.github.io/eks-charts',
-    namespace: 'kube-system',
-    version: '1.5.2',
-    timeout: Duration.minutes(14),
-    values: {
-      clusterName: cluster.clusterName,
-      serviceAccount: {
-        name: 'aws-load-balancer-controller',
-        create: false,
-      },
-    },
-  });
-  albService.node.addDependency(albServiceAccount);
-  albService.node.addDependency(certManager);
```

Comment:
can you remove the variable declaration of `certManager`it breaks the build. But leave the cert manager helm